src/index.ts

```diff
@@ -191,11 +191,20 @@ function parseAddress(address: string): string {
     return address;
 }
 
-function parsePassphrase(passphrase: string): string {
+async function parsePassphrase(passphrase: string): Promise<string> {
     if (_.isUndefined(passphrase)) {
```

Comment:
How about negate condition and switch then and else clause.
We can return passphrase early if it is not undefined.

Reply:
Thanks, I will fix it